packages/pipeline/src/concepts/queries/radar_orderbook_usd_prices.ts

```diff
@@ -0,0 +1,51 @@
+import { Connection } from 'typeorm';
+
+/**
+ * Async function that removes the concepts.radar_orderbook_usd_prices table if it exists
+ * @param connection connection to the db instance
+ */
+export async function deleteTableAsync(connection: Connection): Promise<void> {
+    await connection.query(`drop table if exists concepts.radar_orderbook_usd_prices`);
+}
+
+/**
+ * Async function that creates the concepts.radar_orderbook_usd_prices table
```

Comment:
Can we expand the docstring to describe what this query does and what the combined data means? Thinking about our 6-months-later-selves. Especially the `join... on` clauses. I had to read pretty close and I'm still not 100% sure :)